### DISCUSSION TITLE
Do you need more examples?

### DISCUSSION THREAD

**Discussion by ag0n1k** [ramnes/notion-sdk-py#105] [Show and tell]
Hi!

For a while (about a year) I can't find any time to migrate from notion-py to notion-sdk-py. My stoppers were: there are no so much samples, so I need more time to investigate the way to program my bot, maybe not all methods are covered, etc...

But now notion-py is dead. The api changed and broke the integration, so I have no choice. Migration is done by 6 hours - incredible result. I have the code to update any props, so can update the example. Do the code base need it?

If yes: I'll start a new PR

**Comment by ramnes** [comment]
Hey there! First of all, thank you for willing to help.

Feel free to open a PR "as-is" without spending time on it and I'll tell you if that's the kind of thing I'm interested in or not.

The general idea is still that the more examples the better! I just want them to be simple, not fully-fledged apps. :)
